crates/lib/tests/mocked_qpu.rs

```diff
@@ -273,6 +275,15 @@ mod translation {
                 },
             ))
         }
+
+        async fn get_quantum_processor_quil_calibration_program(
+            &self,
+            _request: Request<GetQuantumProcessorQuilCalibrationProgramRequest>,
+        ) -> Result<Response<QuantumProcessorQuilCalibrationProgram>, Status> {
+            Ok(Response::new(QuantumProcessorQuilCalibrationProgram {
+                quil_calibration_program: "H 0\nCNOT 0 1\n".to_string(),
```

Comment:
Not a big deal, but this might confuse someone running the tests if they see a `H 0` come back when they're expecting `DEFCAL`s etc.

Reply:
Fair point, I put a DEFCAL in there instead.